From d0898592a17d14fc8d48fc5cba71f3d57f0a1856 Mon Sep 17 00:00:00 2001
From: Ben Cooksley <bcooksley@kde.org>
Date: Thu, 30 Dec 2021 07:31:41 +1300
Subject: [PATCH] Use the CDN based endpoint rather than the legacy endpoint
 which hits our download redirector.

CCMAIL: distributions@kde.org

(cherry picked from commit 694e4e24526c5f55e162ad9288782e7c82cac38f)
---
 src/settings/contextmenu/servicemenu.knsrc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/settings/contextmenu/servicemenu.knsrc b/src/settings/contextmenu/servicemenu.knsrc
index 59d459895..4374082c7 100644
--- a/src/settings/contextmenu/servicemenu.knsrc
+++ b/src/settings/contextmenu/servicemenu.knsrc
@@ -1,5 +1,5 @@
 [KNewStuff3]
-ProvidersUrl=https://download.kde.org/ocs/providers.xml
+ProvidersUrl=https://autoconfig.kde.org/ocs/providers.xml
 Categories=Dolphin Service Menus
 TargetDir=servicemenu-download
 Uncompress=never
-- 
GitLab

